### PR TITLE
251 fix/delete posts by url test

### PR DESCRIPTION
**Pull request by rajanit2000**
Its have some failure

### CODE REVIEW

**Comment by mariadanieldeepak** [comment]
@rajanit2000,

Glad that we are unblocked on this now.

Kindly check-in the changes made and verify that the other test cases run as expected. Finally, after a self-review, assign the issue to @sudar for further review.

**Comment by sudar** [comment]
@rajanit2000 

Is this ready for review?

**Comment by rajanit2000** [comment]
Yes @sudar Its ready to review

File: tests/wp-unit/include/Core/Posts/Modules/DeletePostsByURLModuleTest.php [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+		parent::setUp();
+
+		$this->module = new DeletePostsByURLModule();
+	}
+
+	/**
+	 * Helper function.
+	 * create post by URL
+	 */
+	public function get_post_by_url( $url ) {
```
Comment (sudar) [inline tests/wp-unit/include/Core/Posts/Modules/DeletePostsByURLModuleTest.php]: Add more details in PHPDoc for this helper function. We will move it into the test library.

File: tests/wp-unit/include/Core/Posts/Modules/DeletePostsByURLModuleTest.php [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+		return $wp_query->query( $args );
+
+	}
+
+	public function test_to_delete_one_valid_url() {
+		$post = $this->factory->post->create();
+
+		$url = get_permalink( $post );
+
+		//print_r($url);
```
Comment (sudar) [inline tests/wp-unit/include/Core/Posts/Modules/DeletePostsByURLModuleTest.php]: Remove commented code and add documentation for all functions.

**Comment by rajanit2000** [comment]
Done @sudar

**Pull request merged**